src/pynxtools/dataconverter/validation.py

```diff
@@ -755,12 +761,12 @@ def startswith_with_variations(
                     ValidationProblem.UnitWithoutField,
                     not_visited_key.rsplit("/", 1)[0],
                 )
-            if not ignore_undocumented:
-                collector.collect_and_log(
-                    not_visited_key,
-                    ValidationProblem.UnitWithoutDocumentation,
-                    mapping[not_visited_key],
-                )
+            # if not ignore_undocumented:
+            #     collector.collect_and_log(
+            #         not_visited_key,
+            #         ValidationProblem.UnitWithoutDocumentation,
+            #         mapping[not_visited_key],
+            #     )
```

Comment:
This indeed referred to the parent field of an attribute, which gets checked separately. I think we can remove it entirely.